### DISCUSSION TITLE
注册QQ实例时出现错误，这个等待时间可以修改吗

### DISCUSSION THREAD

**Discussion by kumu7y** [Mrs4s/go-cqhttp#2295] [Q&A]
![image](https://github.com/Mrs4s/go-cqhttp/assets/85327198/9cb29074-d35e-4be6-9675-d10003ce7b16)
实测是可以计算出来的就是设备新能可能不够，计算时间较长，这个时间可以延长吗

**Comment by Erstwhile-Liu** [comment]
这个好像并不影响使用，它虽然超时报错，但是请求已经发出去了，实例也注册成功了

**Reply by kumu7y** [reply]
是的，但是如果是第一次登录的话，好像登录不上

**Comment by OREOCODEDEV** [comment]
我在dev分支上直接参照这个没有合并的PR[增加签名服务器请求超时时间](https://github.com/Mrs4s/go-cqhttp/pull/2302)修改再编译一下就好了，应该在gocq团队正式支持签名服务器超时配置项之前可以临时解决你的问题
